### PR TITLE
Add Cilium project idea

### PR DESCRIPTION
**Pull request by xmulligan**
Signed-off-by: Bill Mulligan <bmulligan@linuxfoundation.org>

### CODE REVIEW

File: lfx-mentorship/2023/01-Mar-May/project_ideas.md [hunk 1]
```diff
@@ -23,9 +35,9 @@
 
 - Description: Cluster API Provider GCP (CAPG) enables the creation of Kubernetes clusters in GCP with Cluster API. With increasing adoption of Cluster API (CAPI) in general and of CAPG we want to improve the supportability of CAPG, especially for production environments. The first part of this is to add telemetry/tracing using OpenTelemetry so that we can understand and visualize the flow of reconciliation within the provider. The next part is to add a **pprof** endpoint that can be optionally enabled to enable operations/support users to collect profiling information from a running instances of CAPG.
 - Expected Outcome: This work will enable tracing and profiling of a running instance of CAPG (along with supporting docs) to supports operations/support engineers.
-Recommend Skills: Golang, Kubernetes
-Mentors(s): Carlos Panato (@cpanato), Richard Case (@richardcase)
-Upstream Issue: https://github.com/kubernetes-sigs/cluster-api-provider-gcp/issues/810
+- Recommend Skills: Golang, Kubernetes
+- Mentors(s): Carlos Panato (@cpanato), Richard Case (@richardcase)
+- Upstream Issue: https://github.com/kubernetes-sigs/cluster-api-provider-gcp/issues/810
```
Comment (nate-double-u) [inline lfx-mentorship/2023/01-Mar-May/project_ideas.md:40]: I'm not sure what this change is

**Review by nate-double-u — APPROVED** [review]
Thanks for the proposal @xmulligan!

I'm happy to approve as is, but there is a change I don't understand (listed in line).

File: lfx-mentorship/2023/01-Mar-May/project_ideas.md (hunk 1 above)
Comment (aliok) [reply lfx-mentorship/2023/01-Mar-May/project_ideas.md:40]: @xmulligan was being a good citizen; fixing the formatting issues in that page :)

Things look proper now: https://github.com/cncf/mentoring/blob/27b719d3bd3dc4695c585325f61e649d1cc9c2c1/lfx-mentorship/2023/01-Mar-May/project_ideas.md

/lgtm

Comment (xmulligan) [reply lfx-mentorship/2023/01-Mar-May/project_ideas.md:40]: Yes, I was just trying to fix the formatting. I can break that out into a separate PR if you want

Comment (nate-double-u) [reply lfx-mentorship/2023/01-Mar-May/project_ideas.md:40]: Ah, thanks! The github review preview isn't clear.

**Pull request merged**